src/main/dml/fenixedu-treasury-ui.dml

```diff
@@ -0,0 +1,215 @@
+package org.fenixedu.treasury.domain;
+
+
+/* **************
+ * ACCESS CONTROL
+ * **************
+ */
+ 
+class accesscontrol.PersistentTreasuryFrontOfficeGroup extends .org.fenixedu.bennu.core.domain.groups.PersistentGroup {
```

Comment:
@anilmamede this sort of groups are currently deprecated right? So we can analyze how to remove them in a next iteration?